### DISCUSSION TITLE
Kubernetes kubectl provision yaml

### DISCUSSION THREAD

**Discussion by AntonioSun** [anasoid/docker-jmeter#53] [Q&A] [answered]
Thanks for providing such an advanced Jmeter docker image! Even splitting CSV Data on multi nodes has been provided :+1: .

Now that the docker image has such capability of utilizing splitting CSV Data, I'm wondering if you can share how to spin them up with Kubernetes (`kubectl`)?

I'm new to this, so even a starting pointer will be helpful. thx!

**Comment by anasoid** [comment]
Hello @AntonioSun ,

I will work soon on Kuberneties, may be i can give a full example later, 

If an example with terraform and azure container instance can help you  can give it to you

**Reply by AntonioSun** [reply]
azure container would be most helpful to me, :)

**Reply by anasoid** [reply]
Hello @AntonioSun,

I will create a fork from https://github.com/Azure-Samples/jmeter-aci-terraform with my adapted code.

In the beginning of the next week it will be available, doc is not updated as i was for my use only.

**Comment by anasoid** [comment, marked as answer]
Hello @AntonioSun 
You can see repository https://github.com/anasoid/jmeter-aci-terraform

**Reply by AntonioSun** [reply]
Thanks @anasoid, you are still planning to strip out all the terraform dependencies, and provide a merely azure solution, right?

**Comment by anasoid** [comment]
For the moment no, but why you need to strip out Terraform dependencies ??

**Reply by AntonioSun** [reply]
> why you need to strip out Terraform dependencies

I looked into the code first, and the Terraform parts are those I have no idea, and was afraid that those we don't know will hurt us, :). 

Skimming through the readme again just now, I realized that it might be OK for me to give it a try, if I can get the _"infrastructure provisioned by Terraform"_ part sort out first. 

Anyway, thanks for providing it so fast!
